scss/civicrm/common/_base.scss

```diff
@@ -315,3 +315,23 @@ a.sorting_asc::before {
 a.sorting_desc::before {
   @include fa-icon($font-size-base, $fa-var-sort-desc, $brand-primary);
 }
+
+#alpha-filter {
```

Comment:
why not then a separate partial? we can't cram everything into `_base`

Reply:
Ok, I will fix this